CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing to Headset
+
+Welcome to the Headset community!
+
+## Community
+
+We have two main channels of communication:
+- our Slack workspace: https://tinyurl.com/y7m8y5x4
+- issues on GitHub
+
+If you have any questions on where to start and/or how to proceed, let us know!
+
+## Ways to contribute
+
+We welcome many ways of contributing to Headset:
+- finding bugs and [reporting issues](https://github.com/headsetapp/headset-electron/issues)
+- picking and working on an existing [issue](https://github.com/headsetapp/headset-electron/issues) (the ones tagged
+  "help wanted" are usually a good place to start!)
+- proposing and implementing improvements to Headset
+
+## Setting up the environment
+
+1. Fork the repository on GitHub
+
+2. Clone your fork of the repository
+
+  ```
+  git clone https://github.com/<your-username>/headset-electron.git
+  ```
+
+3. Installation is OS-dependent, so choose the appropriate OS directory:
+  - `windows` for Windows
+  - `darwin` for OSX
+  - `linux` for Linux
+
+    and move to it with `cd <os-directory>`.
+
+4. Install dependencies:
+
+  ```
+  npm install
+  ```
+
+5. Now you can run the app:
+
+  ```
+  npm run production-like
+  ```
+
+You are ready to pick an issue and start working on it.
+
+## Merging your changes
+
+We work in a pull request driven flow. If you don't know how to make a PR,
+[here](https://help.github.com/articles/creating-a-pull-request/) is a GitHub doc on it.
+
+It's best to talk about code and share it early, so you can make a pull request 
+even if you haven't finished - it's much easier to improve once other contributors can
+see the code! Just make sure to write it in the pull request message and ask for help.
+
+To submit a pull request:
```

Comment:
I think that here it should be mentioned that any PR has to be locally tested.
From the root, people have to run two commands:
- `npm run lint <directory>`, where `<directory>` is either the OS that people were working on (`darwin`, `linux`, `windows`), the `test` folder, or the `shared` folder. If people work on multiple OS (or directories), they can run the same `npm run lint` followed by a list of all directories separated by spaces (i.e. `npm run lint linux windows shared`)
- `npm run test:<os>`, where `<os>` is either `darwin`, `linux` or `windows`. If working on multiple OSs, the command has to be run for each OS.

Reply:
great, I'll add a section on testing :)